XpringKit/PayID/PayIDUtils.swift

```diff
@@ -3,11 +3,11 @@ public enum PayIDUtils {
   /// The underlying JavaScript based PayIDUtils.
   private static let javaScriptPayIDUtils = JavaScriptPayIDUtils()
 
-  /// Parse a payment pointer string to a payment pointer object
+  /// Parse the given Pay ID to a set of components.
   ///
-  /// - Parameter paymentPointer: The input string payment pointer.
-  /// - Returns: A  `PaymentPointer` object if the input was valid, otherwise undefined.
-  public static func parse(paymentPointer: String) -> (host: String, path: String)? {
-    return javaScriptPayIDUtils.parse(paymentPointer: paymentPointer)
+  /// - Parameter payID: A PayID to parse.
+  /// - Returns: A set of components parsed from the PayID.
+  public static func parse(payID: String) -> (host: String, path: String)? {
```

Comment:
To conform to the PayID-URI spec, this should return something like this:
`(scheme: String, host: String, account: String)`.

For example, a call to `parse(payID: "payid:nkramer$xpring.money")` should return 
`(scheme: "payid", host: "xpring.money", account: "nkramer")`